design-doc-template.adoc

```diff
@@ -0,0 +1,67 @@
+= <INSERT TITLE HERE>
+:author:            Your Name
+:email:             your.email@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/JBTM[JBTM-XXXX]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/[YYYY-XXXX]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
```

Comment:
We normally use things like zulip for discussions.

Reply:
This is the email of the author (who can be anyone in our community). Having the email here will increase the probability to contact the author even if the Narayana team changes its official social channels.

Reply:
The community at large should be allowed to author proposals, this statement implies that you are limiting it to just the Narayana team.

What will the email address be used for?

Reply:
Sorry, I'm not sure I'm following what you mean :confused: . The email reported here is just a further channel to communicate with the author of the proposal. I'm not sure how I'm limiting it to just the Narayana team.BTW, I changed the email reported above this line of code where `@redhat.com` was used.

Reply:
I have resolved this discussion. Feel free to unresolve it if you need to continue the discussion.